src/stage1/ir.cpp

```diff
@@ -17275,6 +17275,10 @@ static Stage1AirInst *ir_analyze_instruction_import(IrAnalyze *ira, Stage1ZirIns
             ir_add_error_node(ira, source_node,
                     buf_sprintf("unable to find '%s'", buf_ptr(import_target_path)));
             return ira->codegen->invalid_inst_gen;
+        } else if (err == ErrorImportPathMismatch) {
+            ir_add_error_node(ira, source_node,
+                    buf_sprintf("import of '%s' mismatches real path: '%s'", buf_ptr(import_target_path), buf_ptr(&full_path)));
```

Comment:
I think this error is too vague, it should mention the likely cause of case mismatch (or truncated leading spaces?) and should not mention `real path` since that doesn't seem to be what the platform this is most likely to occur on (windows) calls it.

Reply:
I had a hard time to get a good message here.
For now I settled with:
`import of '%s' mismatches actual file path: '%s'. Probably wrong capitalisation.`